### PR TITLE
Match more functions

### PR DESCRIPTION
**Pull request by PikalaxALT**
Continuation of my last pull request in which I refactored ewram.h to make it easier to match the last 50 or so NAKED functions.

### CODE REVIEW

File: src/script_menu.c [hunk 1]
```diff
@@ -895,17 +895,43 @@ bool8 ScriptMenu_CreatePCMultichoice(void)
...
+        sp8[r4++] = GetStringWidthInTilesForScriptMenu(gPCText_SomeonesPC);
+
+    sp8[r4++] = GetStringWidthInTilesForScriptMenu(gPCText_PlayersPC);
+
+    sp8[r4++] = GetStringWidthInTilesForScriptMenu(gPCText_LogOff);
+
+    if (FlagGet(FLAG_SYS_GAME_CLEAR))
+    {
+        sp8[r4++] = GetStringWidthInTilesForScriptMenu(gPCText_HallOfFame);
+    }
```
Comment (GriffinRichards) [inline src/script_menu.c:926]: No braces

File: src/pokenav_before.c [hunk 2]
```diff
@@ -2439,972 +2439,169 @@ void sub_80ED620()
...
+            gPokenavStructPtr->unk304 = 6;
+        }
         break;
     case 6:
-        if (sub_8055870()) return;
-        gPokenavStructPtr->unk304 = 0x4;
+        if (!sub_8055870())
+        {
+            gPokenavStructPtr->unk304 = 4;
+        }
```
Comment (GriffinRichards) [inline src/pokenav_before.c:2551]: No braces

File: src/pokenav.c [hunk 3]
```diff
@@ -1614,244 +1380,55 @@ void LoadTrainerEyesDescriptionLines(void)
...
 {
-    int var0;
+    u32 r5;
+
     if (gPokenavStructPtr->unkD15C == 7)
+    {
         return FALSE;
-
-    if (++gPokenavStructPtr->unk306 > 1)
+    }
```
Comment (GriffinRichards) [inline src/pokenav.c:1390]: No braces

File: src/pokenav_before.c [hunk 4]
```diff
@@ -2439,972 +2439,169 @@ void sub_80ED620()
...
+                gPokenavStructPtr->unk304 = 5;
+                break;
             }
-            else goto label2;
         }
-        else goto label2;
-        break;
-label2:
-        if (gMain.newKeys & B_BUTTON) {
+        if (({gMain.newKeys & B_BUTTON;}))
```
[the remark this answers is not in the document]
Comment (PikalaxALT) [reply src/pokenav_before.c:2511]: TODO: Port JOY_NEW etc.

File: src/pokenav.c [hunk 5]
```diff
@@ -6043,65 +4952,71 @@ void sub_80F66E0(void)
...
+        if (({gMain.newAndRepeatedKeys & DPAD_UP;}) && r4 > 0)
         {
-            r4--;
+            while (r4 > 0)
+            {
+                r4--;
+                if (gPokenavStructPtr->unkBC96[r4] != 0)
+                {
+                    break;
+                }
```
[the remark this answers is not in the document]
Comment (PikalaxALT) [reply src/pokenav.c:4970]: I fear there may be cases where the braces are required to match...

[the remark this answers is not in the document]
Comment (PikalaxALT) [reply src/pokenav.c:4970]: ... never mind lmao

File: src/item_menu.c [hunk 6]
```diff
@@ -1782,547 +1779,167 @@ void sub_80A53F8(void)
...
+            if ((gMain.newAndRepeatedKeys & 0xF0) == DPAD_LEFT)
+            {
+                if ((sPopupMenuSelection >= 2) && sPopupMenuActionList[sPopupMenuSelection - 2] != 8)
+                {
+                    PlaySE(SE_SELECT);
+                    sPopupMenuSelection = MoveMenuCursor3(-2);
+                }
+                break;
+            }
+            if ((gMain.newAndRepeatedKeys & 0xF0) == DPAD_RIGHT)
```
Comment (GriffinRichards) [inline src/item_menu.c:1816]: More instances of `0xF0` for `DPAD_ANY`

**Pull request merged**